### DISCUSSION TITLE
Help needed

### DISCUSSION THREAD

**Discussion by ngrfgt** [terminatorul/NvStrapsReBar#9] [Q&A] [answered]
As far as I can tell I did everything right and triple checked it, but ReBAR still appears to be disabled for me in GPU-Z and Nv profile inspector. I don't know what's wrong so I don't know what to fix. Any help would be appreciated.
![image](https://github.com/terminatorul/NvStrapsReBar/assets/27920734/2f042aa3-eb00-44dd-af5f-e8724035ca93)

**Comment by terminatorul** [comment]
Please show your CPU-Z .txt report, and your modified LocalPciGpu.h header file

**Reply by ngrfgt** [reply]
[CPU-Z .txt](https://github.com/terminatorul/NvStrapsReBar/files/14216847/CPU-Z.txt)
![image](https://github.com/terminatorul/NvStrapsReBar/assets/27920734/ab89fa77-2409-4228-8045-749e94a41be4)

**Comment by terminatorul** [comment, marked as answer]
Please correct the following lines in [LocalPciGpu.h](https://github.com/terminatorul/NvStrapsReBar/blob/master/ReBarDxe/include/LocalPciGPU.h) header:

```C++
#define TARGET_BRIDGE_PCI_VENDOR_ID     0x1022u
#define TARGET_BRIDGE_PCI_DEVICE_ID     0x1483u

#define TARGET_BRIDGE_PCI_BUS           0x00u
#define TARGET_BRIDGE_PCI_DEVICE        0x03u
#define TARGET_BRIDGE_PCI_FUNCTION      0x01u

// Memory range and I/O port range (base + limit) mapped to bridge
// from CPU-Z .txt report of the bridge and GPU

// PCIe config register offset 0x20
#define TARGET_BRIDGE_MEM_BASE_LIMIT  UINT32_C(0xFC00'FB00)                 // Should cover the GPU BAR0

// PCIe config register offset 0x1C
#define TARGET_BRIDGE_IO_BASE_LIMIT   0xE1E1u

```

Build the DXE driver NvStrapsReBar.ffs again, and re-flash your motherboard (remember to re-enable Above 4G Decoding and disable CSM, before you enable ReBAR)

**Reply by ngrfgt** [reply]
Thank you that worked. I was looking at an entirely different PCIe slot which was wrong.
